Change deprecated eye function to new I function

PaulDebus opened the pull request:
Hello,

I just realized, that the `eye` function, that is used in two places, is not supported by new versions of Julia. After 1.0 I think. Just a small fix, two occurrences.

`I` is the replacement, which in this version returns a binary array. It worked as a replacement in the tests I did and the unit tests also passed. If you want, I can convert the result to `Float64` arrays.

Thanks for the great lib!

PaulDebus committed:
change deprecated eye function to new I function

PaulDebus committed:
change deprecated eye function to new I function

peterkovesi commented:
Thank you for that.  I am currently in a tent with only a phone as a computing device. I will be back online on Monday. The package is well overdue for some maintenance. Cheers.

PaulDebus commented:
Nice, enjoy the time! No hurry.

I hope I will do some more using the package, so maybe I do some more!

Have a nice weekend!

PaulDebus committed:
remove interim functions for old Julia versions

PaulDebus committed:
Merge branch 'master' of github.com:PaulDebus/ImageProjectiveGeometry.jl

PaulDebus commented:
So I changed some more and the merge request directly included those changes. 
I replaced some interim functions you created for older Julia versions, 0.6 I guess. As those days are over, I think the new syntax is better. Somehow, this change is already included here. Sorry

peterkovesi commented:
Many thanks for that.  I need to update the Project.toml and incorporate upper bounds on dependencies.  I will then tag and register.  In the meantime there is a mysterious failure in the RANSAC code just for 1.2 on Linux which I will have to investigate.
